src/ocp_postprocess.rs

```diff
@@ -56,13 +59,39 @@ pub(crate) async fn ocp_postprocess(
     .await
     .context("fixing dep annotations for openshift-apiserver")?;
 
+    fix_deployment_spec_hash_annotation(
+        in_memory_etcd_client,
+        K8sResourceLocation::new(Some("openshift-apiserver"), "Deployment", "apiserver", "v1"),
+    )
+    .await
+    .context("fixing spec-hash annotation for openshift-apiserver")?;
+
     fix_deployment_dep_annotations(
         in_memory_etcd_client,
         K8sResourceLocation::new(Some("openshift-oauth-apiserver"), "Deployment", "apiserver", "v1"),
     )
     .await
     .context("fixing dep annotations for openshift-oauth-apiserver")?;
 
+    fix_deployment_spec_hash_annotation(
+        in_memory_etcd_client,
+        K8sResourceLocation::new(Some("openshift-oauth-apiserver"), "Deployment", "apiserver", "v1"),
+    )
+    .await
+    .context("fixing spec-hash annotation for openshift-oauth-apiserver")?;
+
+    // TODO: fix spec-hash for openshift-authentication/oauth-openshift
```

Comment:
Please state reason why it's commented out above the comment

Reply:
That's still WIP, as I have to compute [this](https://github.com/openshift/cluster-authentication-operator/blob/master/pkg/controllers/deployment/default_deployment.go#L54) first. So I will either properly implement it (if I succeed in that computation) or I'll remove this altogether :+1: